tests/test_db_realistic_coverage.py

```
@@ -57,35 +64,23 @@ def test_database_transaction_failures_realistic(self):
             ]
 
             for query in problematic_queries:
-                try:
+                with contextlib.suppress(Exception):
                     result = execute_query(query)
```

Ruff (0.13.3)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)